requirements-test.txt

```diff
@@ -1,8 +1,7 @@
+-r requirements.txt
 mock
-vcrpy==1.10.3
-# FIXME remove the constraint after resolving
-# https://github.com/pytest-dev/pytest/issues/2966
-pytest<3.3.0
-pytest-cov<2.6.0
-pytest-catchlog
-responses==0.10.6
+vcrpy
+pytest
```

Comment:
Based on our experience with a few open source projects, it’s best to freeze pytest, so that CI will not suddenly start failing in `master` due to new checks being added. Instead, we can make it a habit to update it after a release, for example, either addressing or temporarily ignoring any new check added.